seatunnel-core/seatunnel-core-base/src/main/java/org/apache/seatunnel/command/BaseTaskExecuteCommand.java

```diff
@@ -73,6 +74,30 @@ protected final void prepare(E env, List<? extends Plugin<E>>... plugins) {
         }
     }
 
+    /**
+     * Execute close method defined in {@link org.apache.seatunnel.plugin.Plugin}
+     *
+     * @param plugins plugin list
+     */
+    @SafeVarargs
+    protected final void close(List<? extends Plugin<E>>... plugins) {
+        List<RuntimeException> exceptionHolder = new ArrayList<>();
+        for (List<? extends Plugin<E>> pluginList : plugins) {
+            for (Plugin<E> plugin : pluginList) {
+                try (Plugin<?> closed = plugin) {
+                    // ignore
+                } catch (Exception e) {
+                    exceptionHolder.add(new RuntimeException(String.format("plugin %s closed error, errorMessage: %s",
```

Comment:
If you only collect message finally, you don't need to new RuntimeException because it's too expensive.
but I suggest use Suppressed exception feature to keep stacktrace.

Reply:
@yx91490 Thanks for your review, agree with you, use suppressed exception is great. And I think use `RuntimeException` here is accepted, since this method will not execute in high frequency.

Please take a look if there is still anything need to modify.

Reply:
LGTM